Makefile

```diff
@@ -1,6 +1,6 @@
-.PHONY: all clean compile dryRelease update stop checkFormat format api assembleBenchmarkTestRelease assembleUiTestRelease
+.PHONY: all clean compile dryRelease update stop checkFormat format api assembleBenchmarkTestRelease assembleUiTestRelease javadocs
 
-all: stop clean checkFormat compile dryRelease
+all: stop clean javadocs compile
```

Comment:
actually compile (or `./gradlew build`) will run all of the tasks (checkFormat and dryRelease as well), so we were running them twice actuallty. @marandaneto was it on purpose or just and oversight?

Reply:
IIRC not all tasks were run during `build`, we often found javadocs issues only when releasing, not sure about `checkFormat`.

Reply:
yeah javadoc wasn't running, that's why I added it explicitly here since it's a custom task. But other run just fine